components/tools/OmeroJava/test/integration/AbstractServerTest.java

```diff
@@ -285,6 +298,155 @@ public void tearDown() throws Exception {
     }
 
     /**
+     * Cast the map of strings (e.g. the ALL_GROUPS_CONTEXT)
+     * into implicit context, which asks for <String, String>
```

Comment:
this is going to parse like a weird HTML tag, use something like `{@code <String, String>}` for this kind of thing